client/app/bundles/Events/actions/MembersActions.jsx

```diff
@@ -22,3 +23,12 @@ export const fetchMember = (id) => {
     payload: request
   };
 };
+
```

Comment:
When tested here: http://dev-advocacycommons-pr-191.herokuapp.com/events/1/attendances/new#

This isn't working.